File: Flutter/lib/models/venta_detalle.dart
class VentaDetalle {
  final int ventaDetalleID;
  final String ventaID;
  final int productoID;
  final double cantidad;
  final double precioUnitario;
  final String? atributosSeleccionados;

  VentaDetalle({
    required this.ventaDetalleID,
    required this.ventaID,
    required this.productoID,
    required this.cantidad,
    required this.precioUnitario,
    this.atributosSeleccionados,
  });

  factory VentaDetalle.fromJson(Map<String, dynamic> json) => VentaDetalle(
        ventaDetalleID: json['ventaDetalleID'],
        ventaID: json['ventaID'],
        productoID: json['productoID'],
        cantidad: (json['cantidad'] as num).toDouble(),
        precioUnitario: (json['precioUnitario'] as num).toDouble(),
        atributosSeleccionados: json['atributosSeleccionados'],
      );
}

File: Backend/Backend/Models/VentaDetalle.cs
using System.ComponentModel.DataAnnotations;
namespace Backend.Models;
public class VentaDetalle
{
    [Key]
    public long VentaDetalleID { get; set; }
    public Guid VentaID { get; set; }
    public int ProductoID { get; set; }
    public decimal Cantidad { get; set; }
    public decimal PrecioUnitario { get; set; }
    public string? AtributosSeleccionados { get; set; }
}